app/Token.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use LaravelFCM\Message\OptionsBuilder;
+use LaravelFCM\Message\PayloadDataBuilder;
+use LaravelFCM\Message\PayloadNotificationBuilder;
+use FCM;
+use Carbon\Carbon;
+
+class Token extends Model
+{
+    protected $table = 'tokens';
+
+    public function users()
+    {
+        return $this->hasMany(User::class, 'id', 'user_id');
+    }
+
+    public static function sendPhotoNotification($title, $body, $data, $groupArray)
+    {
+        $minutes_per_notification = Setting::getByID('minutes_per_notification_photo')->value;
+        $timeTmp = Carbon::now()->subMinutes($minutes_per_notification);
+
+        $lastTimeSent = LogNotification::getLastRecord();
+
+        if ($lastTimeSent == null or $timeTmp > $lastTimeSent->created_at) {
+            $logNotification = new LogNotification();
+            $logNotification->name = "photo notfication";
+            $logNotification->save();
+            self::buildFcm($title, $body, $data, $groupArray);
+        }
+    }
+
+    private static function buildFcm($title, $body, $data, $groupArray)
```

Comment:
This function would be better to define in helper class

Reply:
It will be better for me to work with FCM in just one model

Reply:
Separet to the Helper class FCMToken.php in Helpers/FCM/(file)